src/main/java/interface_adapters/Dashboard/DashboardState.java

```diff
@@ -1,14 +1,16 @@
 package interface_adapters.Dashboard;
 
-import entities.PortfolioInformation;
+import java.util.ArrayList;
```

Comment:
nice

Reply:
What's nice about it? I don't even remember what the point of that change was. Re-ordering imports I imagine.

Reply:
decoupling from the entity class is a really good addition